layers/+lang/c-c++/config.el

```diff
@@ -47,3 +50,33 @@
 
 (defvar c-c++-default-mode-for-headers 'c-mode
   "Default mode to open header files. Can be `c-mode' or `c++-mode'.")
+
+;; c-c++-lsp-backend variables
+(defvar c-c++-lsp-cache-dir nil
+  "Cache directory. Absolute and relative paths supported.")
+
+(defvar c-c++-lsp-executable nil
+  "Path to cquery/ccls executable (default value assumes it's in the path)")
+
+(defvar c-c++-lsp-project-whitelist nil
+  "A list of project directory patterns for which lsp-c-c++ should be
+initialized. This overrides `c-c++-lsp-project-blacklist'.")
+
+(defvar c-c++-lsp-project-blacklist nil
+  "A list of project root patterns for which lsp-c-c++ shouldn't be
+initialized. `c-c++-lsp-project-whitelist' is checked first, then this,
+if no pattern matches the project root, lsp-c-c++ will be initialized.")
+
+(defvar c-c++-lsp-sem-highlight-method 'font-lock
```

Comment:
I would suggest removing this option and refer users to ccls/cquery wiki pages. Some people think semantic highlighting distracting.